packages/nodes-base/nodes/Twist/MessageConversationDescription.ts

```diff
@@ -227,4 +247,286 @@ export const messageConversationFields = [
 			// },
 		],
 	},
+	/* -------------------------------------------------------------------------- */
+	/*                                messageConversation:getAll                  */
+	/* -------------------------------------------------------------------------- */
+	{
+		displayName: 'Workspace ID',
+		name: 'workspaceId',
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'getWorkspaces',
+		},
+		default: '',
+		displayOptions: {
+			show: {
+				operation: [
+					'getAll',
+				],
+				resource: [
+					'messageConversation',
+				],
+			},
+		},
+		required: true,
+		description: 'The ID of the workspace.',
+	},
+	{
+		displayName: 'Conversation ID',
+		name: 'conversationId',
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'getConversations',
+			loadOptionsDependsOn: [
+				'workspaceId',
+			],
+		},
+		default: '',
+		displayOptions: {
+			show: {
+				operation: [
+					'getAll',
+				],
+				resource: [
+					'messageConversation',
+				],
+			},
+		},
+		required: true,
+		description: 'The ID of the conversation.',
+	},
+	{
+		displayName: 'Additional Fields',
+		name: 'additionalFields',
+		type: 'collection',
+		displayOptions: {
+			show: {
+				operation: [
+					'getAll',
+				],
+				resource: [
+					'messageConversation',
+				],
+			},
+		},
+		default: {},
+		description: 'Other options to set',
+		options: [
+			{
+				displayName: 'Limit',
+				name: 'limit',
+				type: 'number',
+				default: 50,
+				description: 'Limits the number of messages returned.',
+			},
+			{
+				displayName: 'Starting object index',
+				name: 'from_obj_index',
+				type: 'number',
+				default: 0,
+				description: 'Limit messages starting at the specified object index.',
+			},
+			{
+				displayName: 'Ending object index',
+				name: 'to_obj_index',
+				type: 'number',
+				default: 50,
+				description: 'Limit messages ending at the specified object index.',
+			},
+			{
+				displayName: 'Order By',
+				name: 'order_by',
+				type: 'options',
+				default: 'ASC',
+				description: 'The order of the conversations returned one of DESC or ASC.',
+				options:[
+					{
+						name: 'ASC',
+						value: 'ASC',
+					},
+					{
+						name: 'DESC',
+						value: 'DESC',
+					},
+				],
+			},
+		],
+	},
+	/* -------------------------------------------------------------------------- */
+	/*                                messageConversation:get/delete/update       */
+	/* -------------------------------------------------------------------------- */
+	{
+		displayName: 'ID',
```

Comment:
For better user experience, please specify the expected ID to prevent confusion with other params:

![image](https://user-images.githubusercontent.com/44588767/116075653-7d72a200-a693-11eb-9892-ea5dd93ce4f8.png)